src/equations/polytropic_euler_2d.jl

```diff
@@ -135,6 +135,26 @@ end
     return SVector(f1, f2, f3)
 end
 
+# Calculate 2D flux for a single point
```

Comment:
In the comment for the `flux` in the normal direction (i.e. the function above this one) there is a typo. "1D" -> "2D".